Source/URoboSim/Private/Controller/RJointController.cpp

```diff
@@ -290,6 +290,10 @@ void URJointController::Init()
     }
   else
     {
+      for(auto &Joint : GetOwner()->Joints)
+      {
+        DesiredJointState.Add(Joint.Key, 0.f);
```

Comment:
@HoangGiang93 Hi Giang, dieser teil des codes wird nicht funktionieren. In Robotern wie dem PR2 gibt es passive joints (Gripper) die nicht mit einem Motor genutzt werden. Die Bewegung dieser Joins erfolgt durch die kinematische Kette.